### PR TITLE
 update 3.0 print_only_common_headers 

### PR DESCRIPTION
**Pull request by Nitesh639**
#322 Now I'm deleting approx 30 lines in code.

### CODE REVIEW

**Comment by haritrivedi** [comment]
Do we need to test this?

Thanks,
Hari

**Comment by pradeeban** [comment]
@haritrivedi thanks, that will be helpful.

We don't want to break any functionality.

**Comment by pradeeban** [comment]
@Nitesh639 can you bring your dev branch to sync with Niffler's dev branch? By doing that, the test failure in this pull request should be fixed.

**Comment by Nitesh639** [comment]
OK, I will do it today.

File: modules/png-extraction/ImageExtractor.py [hunk 1]
```diff
@@ -468,39 +468,6 @@ def execute(pickle_file, dicom_home, output_directory, print_images, print_only_
     total_length = 0
 
     metas = glob.glob("{}*.csv".format(meta_directory))
```
Comment (pradeeban) [inline modules/png-extraction/ImageExtractor.py:470]: Pls do not delete this bulk of code.

Rather, wrap them under
if CommonHeadersOnly = True:

That means, 
the below code segment will run only when if CommonHeadersOnly = True:

In Suvpar case, we will always run it as if CommonHeadersOnly = false. So we have no problem.

The rationale why this code should not be deleted - I can explain during our next meeting better.

This, and any other code segment you have deleted or plan to delete - make sure just to make them execute when, if CommonHeadersOnly = True:

**Comment by pradeeban** [comment]
Hari and I reviewed the code, and changes are proposed.

**Pull request closed without merging**